image_builder/audiomini/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:focal
+RUN apt-get update
+RUN apt install wget sudo git python3 dosfstools -y 
```

Comment:
It'd be nice to add a comment that dosfstools is for installing `mkfs.vfat`. Otherwise I see us all forgetting why thats there in the not too distant future

Reply:
Resolved with 51025f7